packages/@vue/cli/lib/Generator.js

```diff
@@ -181,9 +181,13 @@ module.exports = class Generator {
       'private',
       'description',
       'author',
+      'main',
+      'unpkg',
+      'files',
       'scripts',
```

Comment:
I would prefer for the scripts to be above the file-related fields (main, files etc.), because as a developer I have to look for those quite a lot.

This is of course subjective - other's feelings about this?

Reply:
Agreed. This is also a well-established best practise imo

Reply:
You guys are absolutely right. 
Great remarks. 
Hope my  fixes are AOK ;)